src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,69 @@
+package parser;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Disabled;
+import shop.Cart;
+import shop.RealItem;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid asterisks in imports, you can setup IDEA to disable auto-collapse:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports